src/Console/Command/FixCommand.php

```diff
@@ -309,6 +315,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $verbosity = $output->getVerbosity();
         $resolver = new ConfigurationResolver();
+        $resolver->getReportFactory()
+            ->setAddAppliedFixers(OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity())
+            ->setIsDecoratedOutput($output->isDecorated())
+            ->setStopwatch($this->stopwatch)
```

Comment:
@keradus May be remove dependency of stopwatch and pass time and memory to method `process` ?